pkg/ddc/thin/transform_config.go

```diff
@@ -0,0 +1,63 @@
+/*
+  Copyright 2022 The Fluid Authors.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+*/
+
+package thin
+
+import (
+	"encoding/json"
+
+	datav1alpha1 "github.com/fluid-cloudnative/fluid/api/v1alpha1"
+)
+
+func (t *ThinEngine) transformConfig(runtime *datav1alpha1.ThinRuntime,
+	profile *datav1alpha1.ThinRuntimeProfile,
+	dataset *datav1alpha1.Dataset, targetPath string) (config Config, err error) {
+	mounts := []datav1alpha1.Mount{}
+	for _, m := range dataset.Spec.Mounts {
+		m.Options, err = t.genUFSMountOptions(m)
+		if err != nil {
+			return
+		}
+		m.EncryptOptions = []datav1alpha1.EncryptOption{}
```

Comment:
Since encryptOptions is added in Options (in function [genUFSMountOptions](https://github.com/cheyang/fluid/blob/8044a7cd1cde19f2f82bed8921d694a928975d8e/pkg/ddc/thin/ufs_internal.go#L71) ) , line `m.EncryptOptions = []datav1alpha1.EncryptOption{}` maybe is unnecessary?

Reply:
The purpose here is to remove  the value of encryptOptions in the config.json

Reply:
How about assigning it as nil directly?

Reply:
I think it's ok